test/TesterInternal/Deployment/AzureSiloTests.cs

```diff
@@ -90,4 +96,13 @@ public IPEndPoint GetIPEndpoint(string endpointName)
 
         public void UnsubscribeFromStoppingNotification(object handlerObject, EventHandler<object> handler) { }
     }
+
+    public class Startup : IStartup
+    {
+        public IServiceProvider ConfigureServices(IServiceCollection svcCollection)
+        {
+            OrleansInternalServices.RegisterSystemTypes(svcCollection);
```

Comment:
Why should this register the Orleans services?  Doesn't svcCollection already have them?